Combat Rules/1_Combat_Rules.txt

```diff
@@ -0,0 +1,392 @@
+==============================
+Combat
+==============================
+
+	Combat starts when either you or an enemy decides they would like to make an 
+attack (or as the GM decrees). During combat, the flow of the game changes so 
+that both player characters and the Non-Player Characters which are run by the
+Game Master each take a turn in a specific order. 
+
+	Combat can end many different ways. Obviously, if all characters on one side
+or the other are dead, then combat ends. But Combat can also end with one side
+retreating and escaping, or if one side surrenders. Not all combat is to the
+death. Once Combat ends, the flow of play returns to the each character doing
+as they like as described in the Basic Rules Section.
+
+===== Starting a Fight =====
+
+== Initiative ==
+
+	When combat starts, all involved characters roll Initiative to determine the 
+order in which they take turns. Initiative is a d100 roll that is modified only 
+by Soldier class passive abilities, and higher numbers take their turn in combat 
+first.
+
+== Stealth Before Combat ==
+
+	Some characters are very sneaky, and often wish to ambush or sneak past 
+enemies. Stealth encounters can often straddle the line between combat and 
+non-combat. When the encounter is time-sensitive, or if the movement of the 
+sneaker is relative to the movment of what else is happening, then stealth can
+require the same turn-based flow as combat. When the sneaking is happening under
+less pressure, it can often skip the rigor of turn-based gameplay.
+
+== Starting Combat From Hiding ==
+
+	If a character succeeds on their stealth skill checks, gets the drop on
+another character, and decides to attack, then the character or characters that 
+were hidden get to take a whole round without the surprised characters getting
+a turn. This is called the Surprise Round.
```

Comment:
May need another sentence to ensure maximum clarity. Technically an added rule here, but it's pretty well tested.